lib/ansible/modules/system/awall.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# (c) 2017, Ted Trask <ttrask01@yahoo.com>
+#
+# This file is part of Ansible
+#
+# This module is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This software is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this software.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['stableinterface'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: awall
+short_description: Manage awall policies
+version_added: "2.4"
+author: Ted Trask (@tdtrask) <ttrask01@yahoo.com>
+description:
+  - This modules allows for enable/disable/activate of I(awall) policies.
+    Alpine Wall (I(awall)) generates a firewall configuration from the enabled policy files
+    and activates the configuration on the system.
+options:
+  name:
+    description:
+      - A policy name, like C(foo), or mutliple policies, like C(foo, bar).
+    required: false
+    default: null
+  state:
+    description:
+      - The policy(ies) will be C(enabled)
+      - The policy(ies) will be C(disabled)
+    required: false
+    default: enabled
+    choices: [ "enabled", "disabled" ]
+  activate:
+    description:
+      - Activate the new firewall rules. Can be run with other steps or on it's own.
+    required: false
+    default: no
+    choices: [ "yes", "no" ]
+'''
+
+EXAMPLES = '''
```

Comment:
For examples please
1) Use a `- name:` rather than a comment
2) Use multi-line YAML

See https://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html#examples-block for more info